package.json

```diff
@@ -30,15 +35,15 @@
     "expo-status-bar": "^1.4.2",
     "expo-updates": "~0.16.4",
     "react": "18.2.0",
-    "react-native": "0.71.3",
+    "react-native": "0.71.6",
     "react-native-ble-manager": "^10.0.0",
     "react-native-cycling-sensors": "https://github.com/shaunmulligan/react-native-cycling-sensors",
     "react-native-flex-grid": "^1.0.0",
     "react-native-gesture-handler": "^2.9.0",
-    "react-native-maps": "^1.4.0",
+    "react-native-maps": "1.3.2",
```

Comment:
why the downgrade here?

Reply:
Expo raises a warning when you do expo start saying it expected x dep with y version. I'm sure in this case it's fine but easier to suppress the warning this way.

Reply:
Yeah makes sense